internal/api/api.go

```diff
@@ -52,6 +52,7 @@ func Run(ctx context.Context, logger zerolog.Logger, store db.Store) {
 	app.Put("/webhooks/:id", webhookController.UpdateWebhook)
 	app.Delete("/webhooks/:id", webhookController.DeleteWebhook)
 	app.Get("/webhooks/signals", webhookController.GetSignalNames)
+	app.Post("/build-cel", webhookController.BuildCEL)
```

Comment:
could you just add a comment here with a link to the CEL project homepage - most people may not know / remember what it is